customer.go

```diff
@@ -0,0 +1,42 @@
+package goshopify
+
+import (
+	"fmt"
+)
+
+const customersBasePath = "admin/customers"
+
+// CustomerService is an interface for interfacing with the customers endpoints
+// of the Shopify API.
+// See: https://help.shopify.com/api/reference/customer
+type CustomerService interface {
+	Count() (int, error)
+}
+
+// CustomerServiceOp handles communication with the product related methods of
+// the Shopify API.
+type CustomerServiceOp struct {
+	client *Client
+}
+
+type customerCountRoot struct {
```

Comment:
It's inspired by this by the way: https://github.com/digitalocean/godo/blob/master/droplets.go#L117-L121